pulumi-new: add page (#14833)
tldr-pages/tldr: pages/common/pulumi-new.md

# pulumi new

> Create a new Pulumi project.
> More information: <https://www.pulumi.com/docs/iac/cli/commands/pulumi_new/>.

- Choose a template interactively:

`pulumi new`

> - Create a project from a specific template (e.g azure-python):

Reviewer: ```suggestion
- Create a project from a specific template (e.g `azure-python`):
```

`pulumi new {{provided-template}}`

- Create a project from a local file:

`pulumi new {{path/to/templates/aws-typescript}}`

> - Create a project from a specific source control location:
> 
> `pulumi new {{https://github.com/user/repo}}`

Reviewer 2: Instructions unclear. Does it accept only github links, or any Git repo, or can I use SVN too?

```suggestion
- Create a project from a Git repository:

`pulumi new {{url}}`
```

Author: Good point. Any git repo will do, private or hosted on some public site.

> - Use the specified secrets provider with the pulumi.com backend:

Reviewer: ```suggestion
- Use the specified secrets provider with the <pulumi.com> backend:
```
We enclose links inside angle brackets to be rendered by our clients.

> - Use the specified secrets provider with the pulumi.com backend:

Reviewer 2: ```suggestion
- Use the specified secrets provider with the Pulumi backend:
```

> - Use the "passphrase" secrets provider with the pulumi.com backend:
> 
> `pulumi new --secrets-provider=passphrase`

Reviewer 3: ```suggestion
- Use the specified secrets provider with the pulumi.com backend:

`pulumi new --secrets-provider={{passphrase}}`
```
